### PR TITLE
added floating labels to input fields in query page

### PR DESCRIPTION
**Pull request by NdibeRaymond**
issue fixed: #232 

Screenshot:
floating_label

### CODE REVIEW

**Comment by NdibeRaymond** [comment]
> @NdibeRaymond How about we keep placeholders empty here to avoid the duplicate content?

@srish there is no placeholder. What we have is a floating label. Exactly like Suchakra suggested. The label is styled to look like a placeholder. When the user starts typing, the label moves up to give way for the input in the input field. That way we meet our design goal and satisfy accessibility issues too

File: frontend/WikiContrib-Frontend/src/query.js [hunk 1]
```diff
@@ -749,64 +749,77 @@ export class Query extends Component {
...
+                                        onChange={e =>
+                                          this.handlChange(
+                                            e.target.name,
+                                            e.target.value,
+                                            index
+                                          )
+                                        }
+                                      />
+                                      <label className="user_label" htmlFor="gerrit_username">
+                                        Gerrit Username
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/query.js:790]: U small in `Username`

Comment (NdibeRaymond) [reply frontend/WikiContrib-Frontend/src/query.js:790]: Ok @srish . I will make the changes and push

**Comment by rammanoj** [comment]
That's great thanks a lot @NdibeRaymond for working on this.

**Pull request merged**